### ISSUE TITLE
Specifying commit/reversion with conf.gem :github

### ISSUE THREAD

**Issue by ashes999** [mruby/mruby#2895]
When I specify `conf.gem :github => ...`, is there a way to specify what commit I want?  (I want to get consistent builds and develop on multiple machines, so the latest version may change under me.)

**Comment by cremno** [comment]
The default `build_config.rb` has an example:
https://github.com/mruby/mruby/blob/13a2cc3e5d27c33db7f4cf06ece4c44a79c79c53/build_config.rb#L19

**Comment by ashes999** [comment]
It's wasn't clear to me that `checksum_hash` is the same thing as the commit hash. Thanks for confirming that it is what I want.

**Comment by cremno** [comment]
Agree. I think just `:commit` would be better (if not the best) but `:checksum_hash` should be kept for backwards compatibility.

**Comment by ashes999** [comment]
Okay. I'll leave this open as a potential "feature enhancement" for future consideration.

**Comment by matz** [comment]
Closing — `conf.gem :github => ...` commit/version pinning is now possible via mrbgems lockfile (`mruby/lockfile.lock`). Reproducible builds across machines work via lockfile commit.
